### DISCUSSION TITLE
Bump to lowest next version

### DISCUSSION THREAD

**Discussion by Starz0r** [renovatebot/renovate#28392] [Request Help] [answered]
### What would you like help with?

None

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

Is it possible to configure Renovate in a way that it selects only the next version in the list? Say one of the libraries in my application's is like this:
```
	v0.3.0
	v0.2.0
	v0.1.5 <--- what actually happens
	v0.1.4
	v0.1.3
	v0.1.2 <--- preferred bump
	v0.1.1 <--- installed
	v0.1.0
```

While reading through the documentation, I didn't come across anything that would let me do anything similar to what I was looking for. Maybe someone knows how to configure it like so, if not I'd like to request it as a feature.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
No, we don't have any way to achieve this. Is it only for some rare cases you'd want to do this?

**Comment by Starz0r** [comment]
Not really, this is just the default I'd prefer to have. I somewhat already have a tool that can kinda do this, but it means making the PR myself, or just committing it by hand.
